ArcadiaParties-SPA/src/app/header/header.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { State } from '../reducers/reducers';
```

Comment:
As I already told @Duveshka, it is code style convention to place external imports before internal imports and separate them with empty line.

Reply:
I've fixed this.